crates/rome_js_analyze/src/analyzers/nursery/use_getter_return.rs

```diff
@@ -0,0 +1,162 @@
+use crate::ControlFlowGraph;
+use rome_analyze::{context::RuleContext, declare_rule, Rule, RuleDiagnostic};
+use rome_console::markup;
+use rome_control_flow::{ExceptionHandlerKind, InstructionKind};
+use rome_js_syntax::{JsGetterClassMember, JsGetterObjectMember, JsReturnStatement};
+use rome_rowan::{AstNode, NodeOrToken, TextRange};
+use rustc_hash::FxHashSet;
+
+declare_rule! {
+    /// Enforces the presence of non-empty `return` statements in getters.
+    ///
+    /// A _getter_ allows defining a property which is dynamically computed.
+    /// Thus, it is desirable that a _getter_ returns a value.
+    ///
+    /// Source: https://eslint.org/docs/latest/rules/getter-return
+    ///
+    /// ## Examples
+    ///
+    /// ### Invalid
```

Comment:
It would be nice to have an example that shows the control flow in action :)

Like, and if/else statement where we miss one of them